frontend/src/app/shared/components/storages/file-picker-base-modal.component.ts/file-picker-base-modal.component.ts

```diff
@@ -132,7 +132,7 @@ export abstract class FilePickerBaseModalComponent extends OpModalComponent impl
     };
   }
 
-  private changeLevel(parent:string|null, crumbs:Breadcrumb[]):void {
+  protected changeLevel(parent:string | null, crumbs:Breadcrumb[]):void {
```

Comment:
```suggestion
  protected changeLevel(parent:string|null, crumbs:Breadcrumb[]):void {
```

Our editor config sets no spaces between typedef union types, can you double check your editor that it applies these rules? Otherwise, we'll keep getting these changed back and forth

Reply:
Interestingly, I changed it, because the build pipe discovered this..